lib/Doctrine/ORM/Configuration.php

```diff
@@ -126,6 +126,33 @@ public function setProxyNamespace($ns)
     }
 
     /**
+     * Returns the umask that controls file permissions for generated proxies.
+     *
+     * @return int
+     */
+    public function getProxyUmask()
+    {
+        return isset($this->_attributes['proxyUmask']) ?
+            $this->_attributes['proxyUmask'] : 0002;
+    }
+
+    /**
+     * @since 2.5.6
+     *
+     * @param int $umask The umask that controls file permissions for generated proxies.
+     *
+     * @throws ORMException
+     */
+    public function setProxyUmask($umask)
+    {
+        if ( ! is_int($umask)) {
```

Comment:
Validation is Duplicated in `Doctrine\Common\Proxy\ProxyGenerator` - Is that a problem?